vault/dendron.topic.refactoring.commands.merge-note.md

```diff
@@ -0,0 +1,22 @@
+---
+id: nxarb351z0kfbl5mkw3arw6
+title: 'Dendron: Merge Note'
+desc: Merge a note into another note and update all backlinks
+updated: 1660043063628
+created: 1660042742775
+---
+
+> **{{fm.title}}** merges the _currently active note_ into another note of your choice. Please make sure the note you want to merge is the active note.
+
+> When prompted to select the destination, you cannot select the source note as the destination.
+
+## Summary
+
+{{fm.desc}}
+
+## Keybindings
+none
+
+## Details
+
+**{{fm.title}}** is a command that lets you merge the content of your active note into another note of your choice. Once the note is merged, all backlinks that the source note had will be updated so that it points to the merge destination, and the source note will be deleted.
```

Comment:
Should we add here how the merge is handled? i.e., source note contents get appended to bottom.

Reply:
makes sense. will add